### PR TITLE
Add "New in Talk 10" section

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: lib/Chat/Changelog/Manager.php [hunk 1]
```diff
@@ -102,6 +112,10 @@ public function getChangelogs(): array {
 			$this->l->t('- Check out the new grid and call view'),
 			$this->l->t('- You can now upload and drag\'n\'drop files directly from your device into the chat'),
 			$this->l->t('- Shared files are now opened directly inside the chat view with the viewer apps'),
+			$this->l->t('New in Talk 10'),
+			$this->l->t('- You can now search for chats and messages in the unified search in the top bar'),
+			$this->l->t('- Spice up your messages with emojis from the emoji picker') . $emojis,
```
Comment (danxuliu) [inline lib/Chat/Changelog/Manager.php:117]: The emoji should be included in the `l->t()` string for proper localisation (for example, for right-to-left languages).

Comment (nickvergessen) [reply lib/Chat/Changelog/Manager.php:117]: I fear they break transifex :-X

Comment (nickvergessen) [reply lib/Chat/Changelog/Manager.php]: Solved

**Review by danxuliu — APPROVED** [review]
Tested and works :+1:

**Pull request merged**